client/OPUNetGameSelectWnd.cpp

```diff
@@ -233,7 +240,7 @@ void OPUNetGameSelectWnd::InitNetLayer()
 }
```

Comment:
Since InitializeNetTransportLayer is called from several places in the code base, I think we may be causing a memory leak on call `app.netTLayer = opuNetTransportLayer`.

I think we should delete the current app.netTLayer if present before setting, but wasn't sure how the underlying code worked.

Reply:
This comment didn't work right, it should be pointing to line 239. Sorry.

Reply:
Maybe we should track this with an issue, showing the relevant section of code.

It may very well be a memory leak.

Mind you, there was a bit of funny business going on with how memory was handled for some of this. Certain objects are cleaned up by the game, and so they need to be allocated from the game heap, rather than the DLL's heap. It's possible the current setup may relate to that, though I suspect you found a valid bug.

Reply:
See Issue #117